cordformation/src/main/kotlin/net/corda/plugins/External.kt

```diff
@@ -0,0 +1,28 @@
+package net.corda.plugins;
+
+import org.gradle.api.tasks.Input
+
+open class External {
+
+    @get:Input
```

Comment:
Those are elements that, while not needed 100% of the time, are required often enough. Not every user is going to come in with a customized image, I think a lot re going to tinker with out of the hub standard image.

With a web container like tomcat, you need ports and volumes. If you want to run a .jar, you need command and ports. Only "privileged" is a bit of a special case, but since we know it happens, why not have it?